src/exports/index.ts

```diff
@@ -5,14 +5,20 @@ export type {
   DeployOptions,
   DeploySentTx,
 } from "../contract-deploy.js";
+export { Contract } from "../contract.js";
 export type {
-  Contract,
   ContractFunctionInteraction,
   IntentAction,
   SendOptions,
 } from "../contract.js";
+export { createEip1193ProviderFromAccounts } from "../createEip1193ProviderFromAccounts.js";
 export * from "../popup.js";
-export type { Account, Wallet } from "../types.js";
+export type {
+  Account,
+  Eip1193Provider,
+  TypedEip1193Provider,
```

Comment:
Use `import { Eip1193Provider, TypedEip1193Provider } from "@nemi-fi/wallet-sdk/internal"` in your project